tests/spec/w3c/headers-spec.js

```diff
@@ -44,6 +52,27 @@ describe("W3C — Headers", () => {
       expect(doc.querySelector(".head h2").textContent).toContain(
         "W3C Editor's Draft"
       );
+      expect(
+        normalizedIncludes(
```

Comment:
`innerText` could be used instead but it requires the document to be visually rendered. The test currently disables rendering by `.display = "none"` and replacing this with `.visibility = "hidden" enables `innerText`, which do you prefer?

Reply:
Why would be need to use `innerText` instead?

Reply:
With `innerText` this can be simpler: `expect(document.getElementById("sotd").innerText).toContain(...)`

Reply:
`.visibility = "hidden"` seems like a good idea, even if we don't need `innerText` here (if https://github.com/w3c/respec/pull/2816#discussion_r399832452 works)

Reply:
Seems we don't need `innerText` anyway!